tests/CommunityToolkit.Aspire.SurrealDb.Tests/AspireSurrealClientExtensionsTest.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Hosting;
+using SurrealDb.Net;
+
+namespace CommunityToolkit.Aspire.SurrealDb.Tests;
+
+public class AspireSurrealClientExtensionsTest(SurrealDbContainerFixture containerFixture) : IClassFixture<SurrealDbContainerFixture>
+{
+    private const string DefaultConnectionName = "db";
+
+    private string DefaultConnectionString =>
+            RequiresDockerAttribute.IsSupported ? containerFixture.GetConnectionString() : "http://localhost:27011";
```

Comment:
I think this needs to be changed to work on Windows, as it's providing just a raw endpoint rather than a formatted connection string.

So rather than `http://...` it should be `Endpoint=...`